### ISSUE TITLE
Mise en place de toile de confiance ? 

### ISSUE THREAD

**Issue by BodySplash** [ordre-des-developpeurs/odd-constitution#10]
étiolement en fonction de la distance
attention au copinage

**Comment by kryzaal** [comment]
Qu'appelles-tu "toile de confiance" ?

**Comment by BodySplash** [comment]
C'est une des idées lancée lors de l'assemblée de samedi.

https://fr.wikipedia.org/wiki/Toile_de_confiance 

Pour tenter de limiter les abus et le copinage, une toile de confiance prenant en compte la distance, et l'âge des liens,  pourrait éventuellement être un outil intéressant quand on cherche à savoir si un membre de l'ordre est plus ou moins recommandé par ses pairs.

**Comment by kryzaal** [comment]
Pourquoi pas, mais je pense qu'il faut d'abord monter quelque chose de léger, un cadre aux futures décisions avant de greffer ce genre de "plugins".

**Comment by BodySplash** [comment]
Oui, ça a été notre conclusion : pourquoi pas dans le futur, pour le moment ce n'est pas nécessaire. L'issue est plus là comme un pense bête.
